packages/language-server/src/node.ts

```diff
@@ -0,0 +1,6 @@
+import * as vscode from 'vscode-languageserver/node';
+import { startLanguageServer } from './server';
+
+const connection = vscode.createConnection(vscode.ProposedFeatures.all);
+
+startLanguageServer(connection);
```

Comment:
Just curious, is having this in its own file setting up to better support non-Node environments?